include/rocksdb/customizable.h

```diff
@@ -0,0 +1,189 @@
+// Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
```

Comment:
I suggest to indicate clearly the possible return value (e.g., ok case and Non-ok case) and what does that means of each function, especially those returns "Status" type. Even they are nearly the same as Configurable, it would be better to copy the comments here.

Reply:
For each method that overrides one in Configurable, I put in a see comment to the method in Configurable.  I am not in favor of every method redefining its arguments and returns from the base.  First, the base method defined the contract on what the arguments mean and what can be returned.  Derived classes must follow this contract.  Second, duplicating the comments in the derived classes means that there it is very hard to make sure the documentation stays up to date as things change.  The "see" is standard practice for doxygen/javadoc comments